JitsiConference.js

```diff
@@ -712,7 +712,8 @@ JitsiConference.prototype.leave = async function(reason) {
     let leaveError;
 
     try {
-        await room.leave(reason);
+        await room.leave(reason)
+            .catch(() => {}); // eslint-disable-line no-empty-function
```

Comment:
Why do you need this? It will break the logic below because onMemberLeft won't be called in that case.